### PR TITLE
added schemas and other fields in application

### PR DESCRIPTION
**Pull request by nileshgupta111**

### CODE REVIEW

File: api_specifications/application-api-0.1.0.yaml [hunk 1]
```diff
@@ -310,6 +310,21 @@ components:
           example: Application 1 of the year 2024
         issuedBy:
           $ref: '#/components/schemas/IssuedBy'
+        applicationRaisedBy:
+          type: string
```
Comment (atulgupta2024) [inline api_specifications/application-api-0.1.0.yaml:314]: Why do I need this? There is already a field - onBehalfOf

File: api_specifications/application-api-0.1.0.yaml [hunk 2]
```diff
@@ -310,6 +310,21 @@ components:
...
+          description: Application Raised by (Party Name of the advocate raising application)
+        representedBy:
+          type: string
```
Comment (atulgupta2024) [inline api_specifications/application-api-0.1.0.yaml:317]: there is a field - createdBy

File: api_specifications/application-api-0.1.0.yaml [hunk 3]
```diff
@@ -310,6 +310,21 @@ components:
...
+          description: Represented by (Advocate Name)
+        partyType:
+          type: string
```
Comment (atulgupta2024) [inline api_specifications/application-api-0.1.0.yaml:320]: why is this required?

File: api_specifications/application-api-0.1.0.yaml [hunk 4]
```diff
@@ -310,6 +310,21 @@ components:
...
+          description: Party Type (Complainant/Respondent)
+        reasonForApplication:
+          type: string
```
Comment (atulgupta2024) [inline api_specifications/application-api-0.1.0.yaml:323]: There is a field - applicationType. Which is to link the master data for types of application. Is that not usable? it this a free flow text?

**Review by atulgupta2024 — CHANGES_REQUESTED** [review]

File: api_specifications/application-api-0.1.0.yaml (hunk 4 above)
Comment (nileshgupta111) [reply api_specifications/application-api-0.1.0.yaml:323]: I created this field to store reason, example  - Reason for Rescheduling, Reason for Extension.  For different application type we have to store the reason so I created this common field to store the reasonForApplication

File: api_specifications/application-api-0.1.0.yaml (hunk 1 above)
Comment (nileshgupta111) [reply api_specifications/application-api-0.1.0.yaml:314]: @atulgupta2024  onBehalfOf is the ID of the users for whom the application has been raised. Application Raised by is the Party Name of the advocate raising application. Is this two same?

**Review by atulgupta2024 — APPROVED** [review]
